Packs/SafeBreach/Integrations/SafeBreach_v2/README.md

```diff
@@ -0,0 +1,690 @@
+
+
+### safebreach-rerun-simulation
+***
+Rerun a specific SafeBreach simulation in your environment
+
+
+##### Base Command
+
+`safebreach-rerun-simulation`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| simulationId | The id of the simulation to rerun | Required | 
+
+
+##### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| SafeBreach.Simulation.Id | Number | Simulation unique id | 
+| SafeBreach.Simulation.Rerun.Name | String | Simulation rerun test name | 
+| SafeBreach.Simulation.Rerun.Id | String | Id of the rerun test | 
+| SafeBreach.Simulation.Rerun.ScheduledTime  | Datetime | Time at which the rerun was triggered | 
+| SafeBreach.Test.Id | String | Id of the test | 
+| SafeBreach.Test.Name | String | Name of the test | 
+| SafeBreach.Test.AttacksCount | Number | Count of the attacks executed in the insight rerun test | 
+| SafeBreach.Test.Status | String | Test run status. For insight rerun \- “PENDING” | 
+| SafeBreach.Test.ScheduledTime | Datetime | Time at which the test was triggered | 
+
+
+##### Command Example
+```!safebreach-rerun-simulation simulationId=d937cd0e5fd4e2c9266801b7bd17e097```
+
+##### Context Example
+```
+{
+    "SafeBreach": {
+        "Simulation": {
+            "Id": "d937cd0e5fd4e2c9266801b7bd17e097",
+            "Rerun": {
+                "Id": "1585820802418.34",
+                "Name": "Rerun (Demisto) - #(3055) Write wannacry malware to disk",
+                "ScheduledTime": "2020-04-02T09:46:42.422750"
+            }
+        },
+        "Test": {
+            "AttacksCount": 1,
+            "Id": "1585820802418.34",
+            "Name": "Rerun (Demisto) - #(3055) Write wannacry malware to disk",
+            "Status": "Pending"
+        }
+    }
+}
+```
+
+##### Human Readable Output
+### SafeBreach Rerun Simualtion
+|Simulation Id|Test Id|Name|
+|---|---|---|
+| d937cd0e5fd4e2c9266801b7bd17e097 | 1585820802418.34 | Rerun (Demisto) - #(3055) Write wannacry malware to disk |
+
+### safebreach-get-indicators
+***
+Fetches SafeBreach Insights and extracts indicators out of them, creating new or updating existing indicators.
+
+
+##### Base Command
+
+`safebreach-get-indicators`
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | Limits the maximal number of generated indicators. Default - 1000. | Optional | 
+| insightCategory | Multi-select option for category of the insights to get remediation data for:<br/>Network Access, Network Inspection, Endpoint, Email, Web, Data Leak | Optional | 
+| insightDataType | Multi-select option for remediation data type to get:<br/>Hash, Domain, URI, Command, Port, Protocol, Registry | Optional | 
+
+
+##### Context Output
+
+There is no context output for this command.
+
+##### Command Example
+```!safebreach-get-indicators limit=10```
+
+##### Context Example
+```
+{}
```

Comment:
the response is empty
Add here a valid json

Reply:
As Francesco suggest we remove any output when calling to get-indicators/fetch-indicators.
Therefore the context is empty and there nothing to specify. 
Should we change it to 'None'?